tmt/steps/__init__.py

```diff
@@ -96,15 +123,16 @@ def __init__(self, data=None, plan=None, name=None, workdir=None):
                     f"of the '{self.plan}' plan.")
 
     @property
-    def enabled(self):
+    def enabled(self) -> Optional[bool]:
```

Comment:
I wonder what's the meaning of `None` as a return value of this property.

@psss would `bool` work here, with `False` being returned in the case of `AttributeError`?

Reply:
The `enabled` property only makes sense if a `Run` instance is attached to the plan which is not the case for example when `tmt plan show` is called (there is no run involved). So if there is no run, which would say if the steps is or is not enabled, we return `None` as the value does not make any sense in this context.

Reply:
I had to re-read the sentence several times, and even after that I still think it's weird :) But ok, let's keep `Optional`, one day I might start digging into what's happening in the deep :)